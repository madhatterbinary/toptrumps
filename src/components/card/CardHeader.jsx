import React from 'react';
import PropTypes from 'prop-types';

const CardHeader = (props) => {
  const {
    title,
    model,
    manufacturer,
  } = props;

  return (
    <div className="card-header">
      <h3>{title}</h3>
      <div className="card-additional-info">
        <div>Model: {model}</div>
        <div>Manufacturer: {manufacturer}</div>
      </div>
    </div>
  );
}

CardHeader.propTypes = {
  title: PropTypes.string,
  model: PropTypes.string,
  manufacturer: PropTypes.string,
}

export default CardHeader;
