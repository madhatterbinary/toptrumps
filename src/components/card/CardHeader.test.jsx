import React from 'react';
import CardHeader from './CardHeader';
import TestRenderer from 'react-test-renderer';

describe('CardHeader', () => {
  it('renders', () => {
    const component = TestRenderer.create(<CardHeader title="" model="" status="" manufacturer="" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
