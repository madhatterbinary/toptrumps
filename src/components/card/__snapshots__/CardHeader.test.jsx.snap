// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader renders 1`] = `
<div
  className="card-header"
>
  <h3>
    
  </h3>
  <div
    className="card-additional-info"
  >
    <div>
      Model: 
      
    </div>
    <div>
      Manufacturer: 
      
    </div>
  </div>
</div>
`;
